server/mdm/maintainedapps/apps.json

```diff
@@ -0,0 +1,97 @@
+[
+	{
+		"identifier": "1password",
+		"bundle_identifier": "com.1password.1password",
+		"installer_format": "zip:app"
+	},
+	{
+		"identifier": "adobe-acrobat-reader",
+		"bundle_identifier": "com.adobe.Reader",
+		"installer_format": "dmg:pkg"
+	},
+	{
+		"identifier": "box-drive",
+		"bundle_identifier": "???",
```

Comment:
Is this (`"???"`) the expected string to use if we don't know the bundle identifier?

Reply:
:D No, good thing to mention indeed, this will need to be figured out, I just couldn't find that app in dogfood to grab the bundle identifier. I'll try to get those filled for the next PR, testing those apps on my mac mini.

Reply:
I added it as a checkbox to my ticket just to make sure I don't forget about it, thanks!